### ISSUE TITLE
can deeplift explain text classification?

### ISSUE THREAD

**Issue by hanluxq** [kundajelab/deeplift#128]

**Comment by AndreaMagnanini** [comment]
Hi Lu Han. I tried to apply this version of DeepLIFT to a cnn for text classification. The problem to solve is the following:
at the moment, in this library there's no implementation for any embedding layer conversion. This makes impossibile to use weights loaded froma a model for those kind of layers.

**Comment by AvantiShri** [comment]
Yes, this codebase is not currently being updated as I have graduated and alternative implementations of deeplift exist - please see corresponding section in the faq titled "my model architecture is not supported by this DeepLIFT implementation. What should I do?"
